L1Trigger/TrackTrigger/src/L1TrackQuality.cc

```diff
@@ -74,19 +74,23 @@ std::vector<float> L1TrackQuality::featureTransform(TTTrack<Ref_Phase2TrackerDig
 
   // get other variables directly from TTTrack
   float tmp_trk_z0 = aTrack.z0();
+  float tmp_trk_z0_scaled = tmp_trk_z0 / abs(aTrack.minZ0);
```

Comment:
Do neural nets only work well if the input variables are all in the range -1 to +1? Why do we need to scale the z0 in this way?

Reply:
In general, performance tends to be better when inputs are in a small range, yes. But the main reason we do this scaling is because z0 has this multiplier applied to it in the track word, and it's not base 2 so it would take more resources than necessary to convert the z0 from the track word back to the original value for the track quality classifier. Instead, we can just use the scaled z0 value that comes from the word, no extra conversion or resources needed.